config/sidekiq_scheduler.yml

```diff
@@ -104,6 +104,11 @@ FacilityLocationVHADriveTime:
   class: Facilities::PSSGDownload
   description: "Download and store drive time bands"
 
+GlobalDowntimeCache:
+  every: "60m"
```

Comment:
Initial thought was to have it run every hour, but open to changing this if a different frequency makes more sense.